var/spack/repos/builtin/packages/perl-net-ssleay/package.py

```diff
@@ -28,6 +29,6 @@ def configure(self, spec, prefix):
             f.writelines(config_answers)
 
         with open(config_answers_filename, 'r') as f:
+            env['OPENSSL_PREFIX'] = self.spec['openssl'].prefix
             inspect.getmodule(self).perl('Makefile.PL', 'INSTALL_BASE={0}'.
-                                         format(prefix), 'OPENSSL_PREFIX=%s' %
-                                         self.spec['openssl'].prefix, input=f)
+                                         format(prefix), input=f)
```

Comment:
Did this work before? Can INSTALL_BASE also be supplied via the environment?

Reply:
I am guessing it did not and just picked up the globally installed OpenSSL.

`INSTALL_BASE` seems to be a proper MakeMaker variable and it supposed to be passed to the Makefile, see: https://perldoc.perl.org/ExtUtils/MakeMaker.html